pkg/kubernetes/views/project-modify.html

```diff
@@ -55,6 +55,13 @@ <h3 class="modal-title" ng-if="project" translatable="yes">Change project</h3>
                 </td>
             </tr>
         </table>
+        <div class="alert dialog alert-info" ng-if="!project">
+          <span class="pficon pficon-info"></span>
+          <span translatable="yes">
+            A project is a namespace for images. It corresponds to a Docker repository.
```

Comment:
I don't think a project corresponds to a docker repository. An imagestream responds to a docker repository.